CMakeLists.txt

```diff
@@ -44,7 +44,6 @@ option(run_unittests "set run_unittests to ON to run unittests (default is OFF)"
 option(skip_samples "set skip_samples to ON to skip building samples (default is OFF)[if possible, they are always built]" OFF)
```

Comment:
Removed.

---
In reply to: [107511461](https://github.com/Azure/azure-uamqp-c/pull/58#discussion_r107511461) [](ancestors = 107511461)